staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -25,14 +25,16 @@ import (
 
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apiserver/pkg/admission"
+	openapinamer "k8s.io/apiserver/pkg/endpoints/openapi"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	genericoptions "k8s.io/apiserver/pkg/server/options"
+	"k8s.io/kubernetes/pkg/generated/openapi"
```

Comment:
should be `"k8s.io/sample-apiserver/pkg/generated/openapi"`

Reply:
oh yes. Fixing.

Reply:
fixed